Failed to setup chat color terminal replacer.

### Describe the bug


After updating to Paper 1.21.3, using Triton v3.11.0 and ProtocolLib v5.4.0-SNAPSHOT-738, the messages in console are not translating properly and their color codes are not parsing at all.

Here's the error printed by Triton on server startup:

```
[13:10:57 ERROR]: [Triton] Failed to setup chat color terminal replacer.
java.lang.NoClassDefFoundError: org/fusesource/jansi/Ansi
        at Triton.jar/com.rexcantor64.triton.terminal.ChatColorTerminalReplacer.<init>(ChatColorTerminalReplacer.java:16) ~[Triton.jar:?]
        at Triton.jar/com.rexcantor64.triton.terminal.TritonTerminalRewrite.<init>(TritonTerminalRewrite.java:19) ~[Triton.jar:?]
        at Triton.jar/com.rexcantor64.triton.terminal.TritonTerminalRewrite.createPolicy(TritonTerminalRewrite.java:27) ~[Triton.jar:?]
        at Triton.jar/com.rexcantor64.triton.terminal.Log4jInjector.injectAppender(Log4jInjector.java:35) ~[Triton.jar:?]
        at Triton.jar/com.rexcantor64.triton.SpigotMLP.onEnable(SpigotMLP.java:110) ~[Triton.jar:?]
        at Triton.jar/com.rexcantor64.triton.plugin.SpigotPlugin.onEnable(SpigotPlugin.java:18) ~[Triton.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:288) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:655) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:604) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:768) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:530) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:326) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1235) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:340) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ClassNotFoundException: org.fusesource.jansi.Ansi
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:197) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:164) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
        ... 18 more
```

### How can the bug be reproduced?

Run the plugin on Paper 1.21.3 and have console translating enabled.

### Expected behaviour

Messages in the terminal should work properly and the error should not be present.

### Which Spigot fork are you using?

PaperMC

### What's your server version?

Paper version 1.21.3-81-master@da71382 (2024-12-03T14:58:13Z) (Implementing API version 1.21.3-R0.1-SNAPSHOT)

### Which proxy are you using?

Velocity

### What's your proxy version?

Velocity 3.4.0-SNAPSHOT (git-965db127-b454)

### What's the MC version of your client?

1.21.4

### What's the Triton version you're using?

3.11.0

### Additional Information

_No response_

--- a/build.gradle
+++ b/build.gradle
@@ -140,4 +140,5 @@ shadowJar {
     relocate 'net.kyori.adventure.text.minimessage', 'com.rexcantor64.shaded.minimessage'
     relocate 'com.tananaev.jsonpatch', 'com.rexcantor64.shaded.jsonpatch'
     relocate 'com.zaxxer.hikari', 'com.rexcantor64.shaded.hikari'
+    relocate 'org.fusesource.jansi', 'com.rexcantor64.shaded.jansi'
 }

--- a/core/build.gradle
+++ b/core/build.gradle
@@ -28,6 +28,9 @@ dependencies {
         exclude group: 'net.kyori', module: 'adventure-text-serialize-plain'
     }
 
+    // Paper 1.21.3+ does not have this anymore, so shade and relocate it
+    implementation 'org.fusesource.jansi:jansi:2.4.0'
+
     compileOnly 'me.clip:placeholderapi:2.11.6'
     compileOnly 'org.apache.logging.log4j:log4j-core:2.17.1'
     implementation 'org.slf4j:slf4j-nop:1.7.36'

